src/client/debugger/jupyter/debuggingManager.ts

```diff
@@ -226,7 +212,19 @@ export class DebuggingManager implements IExtensionSingleActivationService, IDis
     private async startDebugging(doc: NotebookDocument, cell?: NotebookCell, options?: DebugSessionOptions) {
         let dbg = this.notebookToDebugger.get(doc);
         if (!dbg) {
-            dbg = new Debugger(doc, cell, options);
+            const name = cell
+                ? `${path.basename(doc.uri.toString())}?RBL=${cell.index}`
+                : path.basename(doc.uri.toString());
+            const justMyCode = cell ? true : false;
+            const config: DebugConfiguration = {
+                type: DataScience.pythonKernelDebugAdapter(),
+                name: name,
+                request: 'attach',
+                internalConsoleOptions: 'neverOpen',
+                justMyCode: justMyCode,
+                __document: doc.uri.toString()
```

Comment:
Some comments here would be great, as `__document` is not a known property in the debug configuration.